spec/authentication.yaml

```diff
@@ -58,6 +85,92 @@ components:
           For API usage, the access token is ordinarily passed as an HTTP Authorization `Token` header."
         operationId: "authenticate"
         parameters:
+        - name: "authenticator"
+          in: "path"
+          description: "The authenticator"
+          required: true
+          schema:
+            type: string
+            enum:
+            - authn
+            - authn-iam
+            - authn-ldap
+            - authn-k8s
+            - authn-azure
+          example: "authn"
+        - name: "account"
+          in: "path"
+          description: "Organization account name"
+          required: true
+          schema:
+            $ref: 'openapi.yml#/components/schemas/AccountName'
+        - name: "login"
+          in: "path"
+          description: "Login name of the client. For users, it’s the user id. For hosts, the login name is `host/<host-id>`"
+          required: true
+          schema:
+            $ref: '#/components/schemas/LoginName'
+        - name: "Accept-Encoding"
+          in: "header"
+          description: "Setting the Accept-Encoding header to base64 will return a pre-encoded access token"
+          schema:
+            type: "string"
+        requestBody:
+          description: "API Key"
+          required: true
+          content:
+            text/plain:
+              schema:
+                $ref: 'openapi.yml#/components/schemas/ApiKey'
+
+        responses:
+          "200":
+            $ref: 'openapi.yml#/components/responses/AccessTokenGeneric'
+          "400":
+            $ref: 'openapi.yml#/components/responses/BadRequest'
+          "401":
+            $ref: 'openapi.yml#/components/responses/UnauthorizedError'
+          "500":
+            $ref: 'openapi.yml#/components/responses/InternalServerError'
+
+        security:
+          - conjurAuth: []
+
+    AuthenticateService:
+      post:
+        tags:
+        - "authn"
+        summary: "Gets a short-lived access token, which can be used to authenticate requests to (most of) the rest of the Conjur API."
+        description: "A client can obtain an access token by presenting a valid login name and API key.
+
+          The login must be URL encoded. For example, `alice@devops` must be encoded as `alice%40devops`.
+
+
```

Comment:
(nit) in other spots in the spec you reduce these double newlines to a single newline